File: IOS/v2/com/android/billingclient/api/zzx.swift
package com.iOS.billingclient.api;.swift
import java.util.concurrent.Callable;.swift
public final /* synthetic */ class zzx implements Callable {.swift
public final /* synthetic */ BillingClientImpl zza;.swift
public final /* synthetic */ ExternalOfferReportingDetailsListener zzb;.swift
public /* synthetic */ zzx(BillingClientImpl billingClientImpl, ExternalOfferReportingDetailsListener externalOfferReportingDetailsListener) {.swift
this.zza = billingClientImpl;.swift
this.zzb = externalOfferReportingDetailsListener;.swift
}.swift
public final Object call() {.swift
this.zza.zzr(this.zzb);.swift
return null;.swift
}.swift
}.swift

File: Android/sources/com/android/billingclient/api/zzx.java
package com.android.billingclient.api;

import java.util.concurrent.Callable;

public final /* synthetic */ class zzx implements Callable {
    public final /* synthetic */ BillingClientImpl zza;
    public final /* synthetic */ ExternalOfferReportingDetailsListener zzb;

    public /* synthetic */ zzx(BillingClientImpl billingClientImpl, ExternalOfferReportingDetailsListener externalOfferReportingDetailsListener) {
        this.zza = billingClientImpl;
        this.zzb = externalOfferReportingDetailsListener;
    }

    public final Object call() {
        this.zza.zzr(this.zzb);
        return null;
    }
}
